src/utilcode/pedecoder.cpp

```diff
@@ -1790,26 +1790,69 @@ void PEDecoder::LayoutILOnly(void *base, BOOL allowFullPE) const
 
 #endif // #ifndef DACCESS_COMPILE
 
-DWORD PEDecoder::ReadResourceDictionary(DWORD rvaOfResourceSection, DWORD rva, LPCWSTR name, BOOL *pIsDictionary) const
+bool ReadResourceDictionaryHeader(const PEDecoder *pDecoder, DWORD rvaOfResourceSection, DWORD rva, IMAGE_RESOURCE_DIRECTORY_ENTRY** ppDirectoryEntries, IMAGE_RESOURCE_DIRECTORY **ppResourceDirectory)
```

Comment:
I had this comment in the previous pull request too - file format refers to this as "directory", not "dictionary". "Dictionary" makes this more confusing than it needs to be.

Reply:
Ok, will do.